src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor

```diff
@@ -10,15 +10,15 @@
            disabled="@(IsEnabled is false)"
            hidden>
 
-    <div class="bit-checkbox-item bit-checkbox-item__checkbox" @onclick="@HandleCheckboxClick">
-        <i class="bit-checkbox-item__checkmark"></i>
-    </div>
+    <div class="bit-chb-check-wrapper-fluent">
```

Comment:
i think these CSS classes are redundant

Reply:
bit-chb-check-wrapper-fluent was removed and all styles are added directly from parent